packages/components/src/Drawer/Drawer.component.js

```diff
@@ -25,6 +25,15 @@ const STYLES = {
 	exited: { transform: 'translateX(100%)' },
 };
 
+export const SubtitleTagVariants = [
```

Comment:
It's up to you but if the DS changes its Tag variants someday, you would create a breaking change here. That's why creating this in the DS and tying it to the types would be safer.